app/src/main/java/com/example/themealsapp/di/NetworkModule.kt

```diff
@@ -1,37 +1,27 @@
 package com.example.themealsapp.di
 
-import com.example.themealsapp.rest.MealsAPI
-import com.squareup.moshi.Moshi
-import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import com.example.themealsapp.data.remote.MealsAPI
 import dagger.Module
 import dagger.Provides
 import dagger.hilt.InstallIn
 import dagger.hilt.components.SingletonComponent
 import okhttp3.OkHttpClient
 import okhttp3.logging.HttpLoggingInterceptor
 import retrofit2.Retrofit
-import retrofit2.converter.moshi.MoshiConverterFactory
+import retrofit2.converter.gson.GsonConverterFactory
```

Comment:
This change in architecture was reviewed before lecture and maintains functionality of the Room DB implementation. After the code review, it has been discussed and the group chooses to keep it